operator/.downstream_manifests

```diff
@@ -2840,12 +2856,17 @@ spec:
                 type: object
                 x-kubernetes-map-type: atomic
               type:
+                allOf:
```

Comment:
First time seeing the `allOf` think it's from the `type MigrationType string`
does it have some side affect? @mansam wdyt?

Reply:
@mansam thank you!, removed the annotation and it worked as expected and the duplication removed :+1: